packages/test.sh

```diff
@@ -14,7 +14,7 @@ TEST_ENV_DIR=${TEST_ENV_DIR:-$(mktemp -d -t gxpkgtestenvXXXXXX)}
 
 virtualenv -p "$TEST_PYTHON" "$TEST_ENV_DIR"
 . "${TEST_ENV_DIR}/bin/activate"
-pip install mypy
+pip install "mypy==0.790"
```

Comment:
Do we want to change this in dev to install from `lib/galaxy/dependencies/pinned-lint-requirements.txt` to keep the pinning all in the same file?

Reply:
That is a very good idea I think.